cmd/devices-api/api.go

```diff
@@ -198,6 +198,7 @@ func startWebAPI(logger zerolog.Logger, settings *config.Settings, pdb db.Store,
 	vehicleAddr := common.HexToAddress(settings.VehicleNFTAddress)
 
 	// vehicle command privileges
+	vPriv.Patch("/vin", privTokenWare.OneOf(vehicleAddr, []privileges.Privilege{privileges.VehicleCommands}), userDeviceController.UpdateVIN)
```

Comment:
should we create a new privilege for updating the VIN? Like what if the user doesn't have the VehicleCommands privilege, or is `provides access to all commands that can be sent to the device.` one of the default privileges you get when pairing

Reply:
The user has all privileges on his own NFT.